apps/arena/priv/config.json

```diff
@@ -65,11 +65,391 @@
+      },
+      {
+        "position": {
+          "x": 0.0,
+          "y": 0.0
+        },
+        "vertices": [
+          {
+            "x": 3216.0,
+            "y": 1769.0
+          },
+          {
+            "x": 3484.0,
+            "y": 2469.0
+          },
+          {
+            "x": 2251.0,
+            "y": 2300.0
+          },
+          {
+            "x": 2051.0,
+            "y": 2150.0
+          }
+        ],
+        "radius": 0.0,
+        "shape": "polygon"
+      },
+      {
+        "position": {
+          "x": 0.0,
+          "y": 0.0
+        },
+        "vertices": [
+          {
+            "x": -3284.0,
+            "y": 650.0
+          },
+          {
+            "x": -4149.0,
+            "y": 650.0
+          },
+          {
+            "x": -3723.0,
+            "y": 1188.0
+          },
+          {
+            "x": -2900.0,
+            "y": 1474.0
+          },
+          {
+            "x": -2850.0,
+            "y": 1250.0
+          }
+        ],
+        "radius": 0.0,
+        "shape": "polygon"
+      },
+      {
+        "position": {
+          "x": 0.0,
+          "y": 0.0
+        },
+        "vertices": [
+          {
+            "x": -4149.0,
+            "y": 650.0
+          },
+          {
+            "x": -3284.0,
+            "y": 650.0
+          },
+          {
+            "x": -3516.0,
+            "y": -369.0
+          },
+          {
+            "x": -3784.0,
+            "y": -771.0
+          },
+          {
+            "x": -4220.0,
+            "y": -421.0
+          },
+          {
+            "x": -4398.0,
+            "y": 54.0
+          }
+        ],
+        "radius": 0.0,
+        "shape": "polygon"
+      },
+      {
+        "position": {
+          "x": 0.0,
+          "y": 0.0
+        },
+        "vertices": [
+          {
+            "x": -3516.0,
+            "y": -369.0
+          },
+          {
+            "x": -3784.0,
+            "y": -771.0
+          },
+          {
+            "x": -3051.0,
+            "y": -1050.0
+          },
+          {
+            "x": -2751.0,
+            "y": -950.0
+          }
+        ],
+        "radius": 0.0,
+        "shape": "polygon"
+      },
+      {
+        "position": {
+          "x": 0.0,
+          "y": 0.0
+        },
+        "vertices": [
+          {
+            "x": -1248.0,
+            "y": -299.0
+          },
+          {
+            "x": -982.1,
+            "y": -349.6
+          },
+          {
+            "x": -675.05,
+            "y": -840.65
+          },
+          {
+            "x": -341.55,
+            "y": -1225.9
+          },
+          {
+            "x": -647.45,
+            "y": -1474.3
+          },
+          {
+            "x": -1017.75,
+            "y": -1130.45
+          },
+          {
+            "x": -1248.0,
+            "y": -299.0
+          }
+        ],
+        "radius": 0.0,
+        "shape": "polygon",
+        "id": "collider 4 - a"
```

Comment:
Is this "id" necessary? Should we add it to the other obstacles?

Reply:
I thought doing a cheat sheet to edit the colliders could be helpful to later on change individually a section of vectors head on, instead of debugging by changing random values to get the vector I want and then start to work on it. I was starting to do this:
<img width="926" alt="Screenshot 2024-04-29 at 08 48 01" src="https://github.com/lambdaclass/mirra_backend/assets/82987608/5de269c4-03a1-4694-90e3-2ea8a3ed58e2">
But since the colliders needed to be re-done later on I just paused this since it was just and idea and not a formal request. I planned on resume this whenever I had to do the colliders again. But, as I said, the IDs could be deleted and this idea dismissed if you see fit